src/config-schema.ts

```diff
@@ -121,6 +121,11 @@ export const configSchema = {
       },
     },
   },
+  enableStockDispense: {
+    _type: Type.Boolean,
+    _description: 'To enable or disable the deduction of stock during the dispensing process',
+    _default: true,
```

Comment:
To maintain the same behavior by default, this default value should be false.

Reply:
i agree